uiowa/uiowa
# Add file deletion to `umd` command.

Resolves #4314.
Resolves #4492 



# How to test

- Review code changes.
- Check out this branch.
- Complete #4318 by running the `blt uiowa:multisite:delete` command. After running the command, verify that the multisite files directory on each application environment was removed. You can do this by SSH-ing into each application environment that the multisite was deleted from and then CD-ing into `/mnt/gfs/$app.$env/sites/`. Then if you `ls -l`, there should be no multisite directory in the results.
- If there were no issues with #4318, proceed to run the command again for #5222.


- [x] Update docs after merge: https://wiki.uiowa.edu/display/drupaldev/Website+requests+via+sitenow.uiowa.edu
